spec/requests/objects_controller_checksums_spec.rb

```diff
@@ -199,16 +200,16 @@
     context 'when no druids param' do
       context 'when param is empty' do
         it 'body has additional information from the exception if available' do
-          post checksums_objects_url, params: { druids: [], format: :json }, headers: valid_auth_header
+          post checksums_objects_url, params: { druids: [], format: :json }.to_json, headers: post_headers
           expect(response).to have_http_status(:bad_request)
           error_response = JSON.parse(response.body)['errors'].first
-          expect(error_response['detail']).to include('does not match value: , example: druid:bc123df4567')
+          expect(error_response['detail']).to include('druids [] contains fewer than min items')
```

Comment:
Hmmm, curious about this change.

Reply:
came about because openapi is enforcing a minimum number of druids, which is triggering a different response

Reply:
I guess I don't get how this wasn't failing before. I wouldn't expect changing the content-type of the request body to make this start failing.

Reply:
i think there wasn't any enforcement of openapi before at all, so there two failures occurring (content type and druid missing) now